Modules/Quotas/src/Quotas.Infrastructure/Persistence/Database/EntityConfigurations/DatawalletModificationEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.DatawalletModifications;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Backbone.Modules.Quotas.Infrastructure.Persistence.Database.EntityConfigurations;
+
+public class DatawalletModificationEntityTypeConfiguration : IEntityTypeConfiguration<DatawalletModification>
+{
+    public void Configure(EntityTypeBuilder<DatawalletModification> builder)
+    {
+        builder.ToTable(nameof(DatawalletModification), "Synchronization", x => x.ExcludeFromMigrations());
```

Comment:
That's right. 
And this makes me wondering whether this has even been manually tested. Because I don't expect this to work, as we try to reference a table that doesn't exist.